install.sh

```diff
@@ -292,6 +150,7 @@ find_sqlite
 echo "SQLite version for Python is ${SQLITE_VERSION}"
 if [ "$SQLITE_MAJOR_VER" -lt "3" ] || [ "$SQLITE_MAJOR_VER" = "3" ] && [ "$SQLITE_MINOR_VER" -lt "8" ]; then
   echo "Only sqlite>=3.8 is supported"
+  echo "Please install sqlite3 per your OS instructions."
```

Comment:
Do CPython packages generally use the system SQLite

Reply:
I think this was an old mistake of mine during the emergency SQLite vulnerability release. We should be seeing what SQLite is compiled into the system Python instead of testing the system SQLite - though we don't really need to even do that anymore

Reply:
I think we are getting the Python SQLite.  I was questioning the guidance of using OS instructions to address an issue with the version.

`SQLITE_VERSION=$($INSTALL_PYTHON_PATH -c 'import sqlite3; print(sqlite3.sqlite_version)')`

Reply:
Yep. The Python query is at least getting the correct data. However we may not even need that data anymore - but probably better to clean that up in a subsequent PR?

Reply:
Yes, I wasn't concerned about the check itself.  Just the wording of the message being added here.  I'm chatting with Earle to resolve this.

Reply:
Typically on unix platforms that I'm aware of, the sqlite3 cpython library is dynamically linked to the system sqlite3 library. If you compile python yourself then you could use any version you wanted of course - but typically you will use the system one as well (e.g., pyenv uses the system library unless you do something really custom)
I think in most cases the instruction is ok.